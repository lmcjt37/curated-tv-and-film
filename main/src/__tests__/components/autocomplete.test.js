import React from 'react';
import ReactDOM from 'react-dom';

import Autocomplete from '../../components/autocomplete';

import { createRender } from '@material-ui/core/test-utils';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Autocomplete />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('snapshot of initial component', () => {
  let render = createRender();
  const component = render(<Autocomplete />);
  expect(component).toMatchSnapshot();
});
