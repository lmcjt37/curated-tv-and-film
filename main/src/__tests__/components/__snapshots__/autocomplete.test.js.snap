// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of initial component 1`] = `
<div
  aria-expanded="false"
  aria-haspopup="listbox"
  aria-labelledby="downshift-simple-label"
  class="Autocomplete-container-23"
  role="combobox"
>
  <div
    class="MuiInputBase-root-28 Autocomplete-inputRoot-25 MuiInputBase-fullWidth-37"
  >
    <input
      aria-autocomplete="list"
      aria-labelledby="downshift-simple-label"
      autocomplete="off"
      class="MuiInputBase-input-38 Autocomplete-inputInput-26"
      id="downshift-simple-input"
      placeholder="Search..."
      spellcheck="false"
      type="text"
      value=""
    />
  </div>
  <div
    aria-labelledby="downshift-simple-label"
    id="downshift-simple-menu"
    role="listbox"
  />
</div>
`;
